test/chess/store/move_test.exs

```diff
@@ -0,0 +1,98 @@
+defmodule Chess.Store.MoveTest do
+  @moduledoc false
+
+  use Chess.DataCase
+
+  describe "move" do
+    alias Chess.Repo
+    alias Chess.Board
+    alias Chess.Store.Move
+
+    import Chess.Factory
+
+    test "move is valid with a game, a from, and a to" do
+      user = insert(:user, %{email: "link@hyrule.com"})
+      opponent = insert(:user, %{email: "zelda@hyrule.com"})
+
+      game = insert(:game, %{
+        board: Board.default,
+        user_id: user.id,
+        opponent_id: opponent.id,
+      })
+
+      changeset = Move.changeset(%Move{}, %{
+        game_id: game.id,
+        from: %{"file" => 4, "rank" => 1},
+        to: %{"file" => 4, "rank" => 3},
+        piece: %{"type" => "pawn", "colour" => "white"},
+      })
+
+      assert changeset.valid?
+      assert {:ok, _move} = Repo.insert(changeset)
+    end
+
+    test "move is invalid without a game" do
+      changeset = Move.changeset(%Move{}, %{
+        from: %{"file" => 4, "rank" => 1},
+        to: %{"file" => 4, "rank" => 3},
+        piece: %{"type" => "pawn", "colour" => "white"},
+      })
+
+      refute changeset.valid?
```

Comment:
What do you think about verifying where the error is?

```elixir
assert changet.errors["game"]
```

That may not be the exact syntax.